Purchases/Purchasing/StoreKitAbstractions/SubscriptionPeriod.swift

```diff
@@ -0,0 +1,70 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  SubscriptionPeriod.swift
+//
+//  Created by Andrés Boedo on 3/12/21.
+
+import Foundation
+import StoreKit
+
+@objc public class SubscriptionPeriod: NSObject {
+
+    public let value: Int
+    public let unit: PeriodUnit
+
+    init(value: Int, unit: PeriodUnit) {
+        self.value = value
+        self.unit = unit
+    }
+
+    @objc public enum PeriodUnit: Int {
+
+        case unknown = -1
+        case day = 0
+        case week = 1
+        case month = 2
+        case year = 3
+
+        @available(iOS 11.2, macOS 10.13.2, tvOS 11.2, watchOS 6.2, *)
+        fileprivate static func from(sk1PeriodUnit: SK1Product.PeriodUnit) -> Self {
```

Comment:
Nit: maybe move these `fileprivate` methods to an extension? That way at first glance we just see the public interface without implementation details.